.storybook/preview.js

```diff
@@ -0,0 +1,9 @@
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
+  controls: {
+    matchers: {
+      color: /(background|color)$/i,
+      date: /Date$/,
+    },
+  },
```

Comment:
```suggestion
  },
  viewMode: 'docs'
```

nit) viewMode: 'docs' 설정 넣는건 어떠신가요?

Reply:
좋은 의견인것 같아요!